.ci/scripts/install-tools.bat

```diff
@@ -16,22 +16,30 @@ mkdir %WORKSPACE%\bin
 
 REM If 32 bits then install the GVM accordingly
 IF NOT EXIST "%PROGRAMFILES(X86)%" (
-    curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-386.exe
+    curl -L -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-386.exe
 )
 
-where /q gvm
+set GVM_BIN=gvm
+where /q %GVM_BIN%
 IF ERRORLEVEL 1 (
+    set GVM_BIN=gvm.exe
     IF EXIST "%PROGRAMFILES(X86)%" (
-        curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-amd64.exe
+        curl -L -o %WORKSPACE%\bin\%GVM_BIN% https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-amd64.exe
     ) ELSE (
-        curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-386.exe
+        curl -L -o %WORKSPACE%\bin\%GVM_BIN% https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-386.exe
     )
     IF ERRORLEVEL 1 (
         exit /b 1
     )
-    dir "%WORKSPACE%\bin" /b
+    if EXIST %WORKSPACE%\bin\%GVM_BIN% (
+        %GVM_BIN% version
+    ) else (
+        REM gvm.exe has not been installed for some unknown reasons
+        exit /b 1
+    )
 )
-FOR /f "tokens=*" %%i IN ('"gvm.exe" use %GO_VERSION% --format=batch') DO %%i
+
+FOR /f "tokens=*" %%i IN ('"%GVM_BIN%" use %GO_VERSION% --format=batch') DO %%i
```

Comment:
is `use ` the correct format for the gvm?

https://github.com/andrewkroh/gvm#installation vs https://github.com/moovweb/gvm#installing-go

Reply:
yes, it's there https://github.com/andrewkroh/gvm/blob/master/cmd/gvm/use.go